### DISCUSSION TITLE
856853 Could not find expected browser (chrome) locally

### DISCUSSION THREAD

**Discussion by sloppy823** [jef/streetmerchant#2179] [Help]
something bad happened, resetting streetmerchant in 5 seconds Could not find expected browser (chrome) locally. Run `npm install` to download the correct Chromium revision (856583).
I tried npm install but it didn't work. I put chrome driver in the folder but it still didn't work how do I fix it?

**Comment by Rchimedes** [comment]
I am running into the same issue. I tried installing puppeteer and chromium via NPM and it said they were already up to date. Do we need to install that specific revision?

**Reply by Reimans** [reply]
You know? I fix mine UNINSTALLING the god d*mn "Node.js built 15" and installing the 14. After deleting the git "streetmerchant" and retrying everything again, i just had to do another step (which was update the NPM in the console [THE CONSOLE WILL GIVE YOU THE COMMAND after installing AND Run it for first time]) and works fine!

**Reply by Rchimedes** [reply]
Nice! I will give this a shot. Thank you!

**Reply by etwibior** [reply]
Thanks reimans.
